server/src/controllers/tagController.js

```diff
@@ -12,21 +12,21 @@ const addJobTags = (req, res) => {
       id: job.id
     }
   })
-    .then( (foundJob) => {
-      tagsData.forEach((tag) => {
-        Tag.findOrCreate({ 
-          where: { name: tag }
-        })
-        .spread((tag, created) => {
-          //Sequelize association that adds job and tag to JobTag join table
-          tag.addJob(foundJob)
-          res.end();
-        })
+  .then((foundJob) => {
+    tagsData.forEach((tag) => {
+      Tag.findOrCreate({ 
+        where: { name: tag }
+      })
+      .spread((tag, created) => {
+        //Sequelize association that adds job and tag to JobTag join table
+        tag.addJob(foundJob)
+        res.end();
       })
     })
-    .catch((err) => {
-      throw err;
-    });
+  })
+  .catch((err) => {
+    throw err;
```

Comment:
It feels weird to "throw" the error after catching it. I think the best practice here is to log the error. The sequelize API "throws" the error, our backend "catches" it, and then we log it to debug later.